test/framework/easyconfig.py

```diff
@@ -2211,7 +2211,7 @@ def test_dump_extra(self):
             '',
         ])
 
-        handle, testec = tempfile.mkstemp(prefix=self.test_prefix, suffix='.eb')
+        handle, testec = tempfile.mkstemp(suffix='.eb')
```

Comment:
Why not change this to `dir=self.test_prefix`, so we're still using the tmpdir that's specific to this test?
Same below

Reply:
Because at the same time that `self.test_prefix` is set the temp directory is set too, so `mkstemp` will use that already. The same is used in other places already, so I think this is a left-over from a change some time ago.